src/anbox/platform/sdl/platform.cpp

```diff
@@ -370,6 +315,49 @@ void Platform::process_input_event(const SDL_Event &event) {
 #endif
 }
 
+bool Platform::calculate_touch_coordinates(const SDL_Event &event,
+                                                 std::int32_t &x,
+                                                 std::int32_t &y) {
+  std::int32_t rel_x = 0;
+  std::int32_t rel_y = 0;
+
+  SDL_Window *window = nullptr;
+
+  window = SDL_GetWindowFromID(focused_sdl_window_id_);
```

Comment:
It's unfortunate that we can't get the window id from the event .. however if this approach works well lets go this way.

Reply:
yes it's unfortunate, the SDL team see the touch device as something more general of a touchscreen, so they don't want to set the windowID through input touch.
on my systems this approach work really well